app/pb_kits/playbook/pb_date_stacked/_date_stacked.jsx

```diff
@@ -2,8 +2,8 @@
 
 import React from 'react'
 import classnames from 'classnames'
+import moment from 'moment'
```

Comment:
We shouldn't be directly importing `moment` here. Originally, the intent was to make use of `dateTime.js` which imports the lib.